tests/views_widgets/Tribe/Events/Views/V2/Widgets/Admin/Widget_List_FormTest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+namespace Tribe\Events\Views\V2\Views\Widgets\Admin;
+
+use Spatie\Snapshots\MatchesSnapshots;
+use Tribe\Events\Views\V2\Widgets\Widget_List;
+use Tribe\Test\Products\WPBrowser\Views\V2\ViewTestCase;
+
+class Widget_List_FormTest extends ViewTestCase {
+
+	use MatchesSnapshots;
+
+	public function setUp() {
+		parent::setUp();
+		\Tribe__Rewrite::instance()->setup();
+	}
+
+	/**
+	 * @test
+	 */
+	public function test_with_default_arguments() {
+		$widget = new Widget_List();
+		$instance = [];
+
+		add_filter(
+			"tribe_widget_{$widget->get_registration_slug()}_arguments",
+			static function ( array $arguments ) use ( $instance ) {
+				return wp_parse_args(
+					$instance,
+					$arguments
+				);
+			}
+		);
+
+		$arguments = $widget->get_arguments();
+		$html = $widget->get_admin_template()->template( 'widgets/list', $arguments, false );
```

Comment:
This use highlights a missing API, the Widget should have a `get_admin_html` method that would enclose this logic and would not require client code to know how the Widget works internally (by feeding `$arguments` to a `$template`.

Reply:
This is now added.